Quadriga/src/main/webapp/WEB-INF/views/public/navbar.jsp

```diff
@@ -1,10 +1,13 @@
 <li ${currentPage == "home" ? "class=\"active\"" : ""}><a
-        href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
+	href="${pageContext.servletContext.contextPath}/sites/${project.unixName}">Home</a></li>
 <li ${currentPage == "networks" ? "class=\"active\"" : ""}><a
```

Comment:
This entry should be active when the current page is your blog, not the networks.